### ISSUE TITLE
Nginx with ssl gives CRSF error - improvement

### ISSUE THREAD

**Issue by paladium** [getsentry/onpremise#550]
Hi everyone, I tried installing sentry on premise without SSL and didn`t have any issues. After I have added the SSL configuration to Nginx:
```nginx
server{
listen 80 ssl;
ssl_certificate ./cert.crt;
ssl_certificate_key ./cert.key;
...
}
```
And in sentry.conf.py uncommented these lines:
```py
SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
SESSION_COOKIE_SECURE = True
CSRF_COOKIE_SECURE = True
```
I was getting: ***CSRF Failed: CSRF token missing or incorrect*** in UI. After digging in the forum, I found that default Nginx configuration needs to be altered slightly. Namely, the line:
```nginx
proxy_set_header Host $host;
```
Needs to be changed to:
```nginx
proxy_set_header Host $http_host;
```
And everything seems to work. I suggest adding it to documentation of SSL for sentry setup or changing the default Nginx configuration.
Thank you.

**Comment by BYK** [comment]
Thanks a lot for taking the effort for sharing this @paladium. Took note for the future "How to run w/ SSL docs".

**Comment by BYK** [comment]
This should be addressed via #534.

**Comment by BYK** [comment]
Closing as we now recommend terminating SSL/TLS at load balancer level, instead of modifying the built-in nginx config: https://develop.sentry.dev/self-hosted/#productionalizing
